var/spack/repos/builtin/packages/dftbplus/package.py

```diff
@@ -62,16 +76,38 @@ class Dftbplus(MakefilePackage):
         default=False,
         description="Use DftD3 dispersion library " "(if you need this dispersion model)",
     )
+    variant(
+        "api",
+        default=True,
+        description="Build the API library " "(if you need to link to DFTB+ from other codes)",
+        when="build_system=cmake",
+    )
+    variant(
+        "openmp",
+        default=True,
+        description="Build with OpenMP support " "(if you need to use OpenMP parallelization)",
+    )
+    variant(
+        "sharedlibs",
+        default=False,
+        description="Build as shared library",
+    )
 
     depends_on("lapack")
     depends_on("blas")
     depends_on("scalapack", when="+mpi")
     depends_on("mpi", when="+mpi")
     depends_on("elsi", when="+elsi")
     depends_on("magma", when="+gpu")
-    depends_on("arpack-ng", when="+arpack")
+    depends_on("arpack-ng", when="+arpack~mpi")
```

Comment:
(I don't know what arpack-ng is/does) Is it in principle possible to use arpack and mpi together?